tests/test_early_stopping.py

```diff
@@ -44,10 +44,11 @@ def test_is_improvement():
 class MockEvaluator(Evaluator):
     """A mock evaluator for testing early stopping."""
 
-    def __init__(self, losses: Iterable[float]) -> None:
+    def __init__(self, losses: Iterable[float], automatic_memory_optimization: bool = True) -> None:
         super().__init__()
         self.losses = tuple(losses)
         self.losses_iter = iter(self.losses)
+        self.automatic_memory_optimization = automatic_memory_optimization
```

Comment:
delete

Reply:
Done in 9a6a26f0086974c9aeb924a4d0187d424664d23b.